devfile.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: 1.0.0
+metadata:
+  generateName: che-devfile-registry-
+
+components:
+  - alias: che-devfile-registry-prod
+    image: 'quay.io/eclipse/che-devfile-registry:nightly'
+    mountSources: true
+    endpoints:
+      - name: http
+        port: 8080
+    command:
+      - sleep
+      - infinity
+    memoryLimit: 512Mi
+    type: dockerimage
+    volumes:
+      - name: images
+        containerPath: /usr/local/apache2/htdocs/images
+
+  - alias: dev
+    image: 'quay.io/eclipse/che-plugin-registry-dev:latest'
+    mountSources: true
+    memoryLimit: 1G
+    type: dockerimage
+
+  - id: redhat/vscode-xml/latest
+    type: chePlugin
+
+  - id: redhat/vscode-yaml/latest
+    type: chePlugin
+
+  - id: eamodio/gitlens/latest
+    type: chePlugin
+    registryUrl: 'https://che-plugin-registry-main.surge.sh/v3'
```

Comment:
I think this nightly plug-in registry is to use that devfile in workspaces.openshift.com ?
probably there it should use the version of the plug-in embedded in the che version of CRW and not the nightly that may not work

Reply:
to me, having issues with these plugins is not a big deal. as long as it won't stop the developer to work on devfile registry.